### ISSUE TITLE
Update Product Profile

### ISSUE THREAD

**Issue by RakshindaAslam** [department-of-veterans-affairs/va.gov-team#56334]
## Issue Description
VA Platform product profile for 526 is incomplete. It needs to be updated with current information and other two older profiles for BDD and Original Claims need to be removed/archived. This will ensure that there is one accurate product profile for [Disability Comp Application (21-526 EZ)](https://depo-platform-documentation.scrollhelp.site/getting-started/disability-comp-application-21-526-ez).

---
## Tasks
- [ ] Update fields for restricted and non-restricted sections of the profile

## Acceptance Criteria
- [ ] Product Profile for Disability Comp Application (21-526 EZ) reflects accurate information.
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by navaeddie** [comment]
@RakshindaAslam @alatempa24  

**TL:DR; We should close this one.** The [Product Profile](https://depo-platform-documentation.scrollhelp.site/getting-started/disability-comp-application-21-526-ez) is essentially up to date. Further updates are blocked by another team who probably want to focus on other priorities.

You may recall, the Product Profile pages request 2 different types of forms to fill them out: Restricted Attributes forms and Unrestricted Attributes forms.

The changes I made via Unrestricted Attributes forms populated very easily.

The changes I made via Restricted Attributes forms have not populated. I contacted someone on the Platform/Tech Team 4 who said they'd improve the update process and make sure our changes were populated, but it did not seem like a high priority item for them.

The Product Profile for 526 is also pretty well populated and I don't think that following through on the last few "Restricted Attributes" would add much value. Those last few changes do not include a lot of important information, and I'm not sure who or how often people refer to those Product Profiles for info.

I could contact my POC on Platform/Tech Team 4 again, but I rather not bother them unless we are sure further Product Profile updates will bring someone value. I recommend we mark this issue as done and closed.
